docs/usage.md

```diff
@@ -52,6 +52,12 @@ Some options are **global only**, and go directly under the handler's name.
 
     More details at [Finding modules](#finding-modules).
 
+- `load_external_modules`: this option allows solving aliases from any external module.
+  In a similar manner as `preload_external_modules`, but from any module.  
+
+  It is recommended to use `preload_external_modules`, and use this option only when
+  `preload_external_modules` is not enough.
```

Comment:
```suggestion
- `load_external_modules`: this option allows resolving aliases to any external module,
  i.e. modules that are directly collected/rendered with autodoc instructions (`:::`).
  Enabling this option will tell the handler to follow imports
  that are made public through the `__all__` variables, recursively.
  **Use with caution:** this can load a *lot* of modules, slowing down your build
  or triggering errors that we do not yet handle.
  **We recommend using the `preload_modules` option instead**,
  which acts as an include-list rather than as include-all.
```